tests/operators/test_ageofair.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2024 Met Office and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests age of air operators."""
+
+import iris
+import iris.cube
+import numpy as np
+import pytest
+
+import CSET.operators.ageofair as ageofair
+
+
+def test_calc_dist():
+    """
+    Test distance calculated from calc_dist in age of air calculation.
+
+    Allow a tolerance of 20km (TBD, expect some error).
+    """
+    # London and Johannesburg coordinates to 2 decimal places.
+    london_coords = (51.51, -0.13)
+    johanbg_coords = (-26.21, 28.03)
+
+    dist = ageofair._calc_dist(london_coords, johanbg_coords)
+    actual_distance = 9068670  # Air line according to Google?!
+
+    assert np.allclose(dist, actual_distance, rtol=1e-06, atol=20000)
+
+
+def get_xwind() -> iris.cube.Cube:
+    """Get regridded xwind to run tests on."""
+    return iris.load_cube("tests/test_data/ageofair/aoa_in_rgd.nc", "x_wind")
+
+
+def get_ywind() -> iris.cube.Cube:
+    """Get regridded ywind to run tests on."""
+    return iris.load_cube("tests/test_data/ageofair/aoa_in_rgd.nc", "y_wind")
+
+
+def get_wwind() -> iris.cube.Cube:
+    """Get regridded wwind to run tests on."""
+    return iris.load_cube(
+        "tests/test_data/ageofair/aoa_in_rgd.nc", "upward_air_velocity"
+    )
+
+
+def get_geopot() -> iris.cube.Cube:
+    """Get regridded geopotential height to run tests on."""
+    return iris.load_cube(
+        "tests/test_data/ageofair/aoa_in_rgd.nc", "geopotential_height"
+    )
+
+
+def test_aoa_noincW_nocyclic(
+    xwind=get_xwind, ywind=get_ywind, wwind=get_wwind, geopot=get_geopot
+):
+    """Test case no vertical velocity and not cyclic."""
+    assert np.allclose(
+        ageofair.compute_ageofair(
+            xwind(), ywind(), wwind(), geopot(), plev=500, incW=False, cyclic=False
+        ).data,
+        iris.load_cube("tests/test_data/ageofair/aoa_out.nc").data,
+        rtol=1e-06,
+        atol=1e-02,
+    )
```

Comment:
```suggestion
def test_aoa_noincW_nocyclic(
    get_xwind, get_ywind, get_wwind, get_geopot
):
    """Test case no vertical velocity and not cyclic."""
    assert np.allclose(
        ageofair.compute_ageofair(
            get_xwind, get_ywind, get_wwind, get_geopot, plev=500, incW=False, cyclic=False
        ).data,
        iris.load_cube("tests/test_data/ageofair/aoa_out.nc").data,
        rtol=1e-06,
        atol=1e-02,
    )
```
Usage of fixtures. You might also want to rename the `get_?wind` functions to just `?wind`, as that is what they are when used.